<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-40</y>
      <width>1001</width>
      <height>931</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Desktop/PROJET-C+.png</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="pdf">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>771</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre des client (anuuelle)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mail">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_nc">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>230</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>230</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_avocat">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>751</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de clients (mois)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mid">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mp">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_me">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mn_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>381</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Tri</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prenom</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_tri">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>90</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>751</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prenom</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="le_rech">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>441</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tb_rech">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>731</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_rech">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>110</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>PDF</string>
     </attribute>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Taxes</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tax">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>381</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_cal">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>681</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
